### ISSUE TITLE
Model of held item Mismatch Upon Respawn

### ISSUE THREAD

**Issue by Drathamus** [MSRevive/MSCScripts#228]
I'm sure this is an ancient bug that really doesn't affect gameplay, but here it is nonetheless. Sometimes upon death, items held will change models. 

Best I've been able to replicate is the Raven Mace turning into.... whatever this is:

https://steamuserimages-a.akamaihd.net/ugc/5102047032333137677/22B8765FFF62CEF9FEECB35D0EF483C9237B9308/?imw=5000&imh=5000&ima=fit&impolicy=Letterbox&imcolor=%23000000&letterbox=false

**Comment by greatguys1** [comment]
I don't think this is a script issue, but I'll leave this here until we have more information about it
